src/UniswapV3VolatilityOracle.sol

```diff
@@ -106,6 +125,24 @@ contract UniswapV3VolatilityOracle is IUniswapV3VolatilityOracle, Keep3rV2Job {
         return timestamp;
     }
 
+    /// @inheritdoc IUniswapV3VolatilityOracle
+    function setDefaultFeeTierForTokenPair(address tokenA, address tokenB, UniswapV3FeeTier tier)
+        external
+        requiresAdmin(msg.sender)
```

Comment:
Is there a case where `requiresAdmin()` needs to take an address which is not `msg.sender`? If not, I think we could just put that in the `requiresAdmin` modifier itself.